test/integration/tests/hash.sh

```diff
@@ -72,7 +72,7 @@ echo "T0naX0u123abc" > $hash_in_file
 tpm2_hash -a p -G sha256 -Q -o $hash_out_file -t $ticket_file < $hash_in_file
 
 expected=`sha256sum $hash_in_file | awk '{print $1}'`
-actual=`cat $hash_out_file | xxd -p -c 256`
```

Comment:
hmm, that's correct. How did this didn't fail?

Reply:
```
$ cat hash.dat | xxd -p -c 256 
fa53f49e265b52890e3a7d36b911ec5cef503d655d48ed6c74ae47da9bde80d2
wcrobert@wcrobert-MOBL1:~$ 
wcrobert@wcrobert-MOBL1:~$ cat hash.dat | xxd -p -c 32
fa53f49e265b52890e3a7d36b911ec5cef503d655d48ed6c74ae47da9bde80d2
```
per the man page:
```
       -c cols | -cols cols
              format  <cols> octets per line. Default 16 (-i: 12, -ps: 30, -b:
              6). Max 256.
```

Reply:
@martinezjavier @williamcroberts 


Seems that it's a bit strange behavior with `xxd`:
![selection_008](https://user-images.githubusercontent.com/22025908/47358239-e1d2d600-d6d1-11e8-951e-6c8cd13b8c0a.png)

As you see `xxd` adds spaces after just after the actual value.

My version:
```
# xxd --help
BusyBox v1.28.3 (2018-10-12 16:01:33 EEST) multi-call binary.

Usage: xxd [OPTIONS] [FILE]

Hex dump FILE (or stdin)

	-g N		Bytes per group
	-c N		Bytes per line
	-p		Show only hex bytes, assumes -c30
	-l LENGTH	Show only first LENGTH bytes
	-s OFFSET	Skip OFFSET bytes

```